internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_up_mgr.go

```diff
@@ -364,7 +367,10 @@ type instanceTypeConsumptionSummaryCalculator struct {
 }
 
 // Calculate returns a instanceTypeConsumptionSummary containing a consumption
-// summary for the provided supportedInstanceTypeLocator
+// summary for the provided supportedInstanceTypeLocator.
+// NOTE - only supportedInstanceTypeLocator instances with clusterType = "managed"
```

Comment:
I think this note is not really accurate in the context of this method.

The `Calculate` method will only take into account clusters whose locator matches. It can be used for any kind of clusters. It happens to be that in the context of scaleup mgr the only locators we set set the cluster type to `managed`  but that's independent.

As I mentioned in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1483/files#r1067898699 I think we should update the documentation of the `supportedInstanceTypeLocator` type itself to document that now it includes the cluster type